makerdao/governance-portal-v2
# Remove breakdown tooltip + align proxy text

### Link to Clubhouse story

https://app.shortcut.com/dux-makerdao/story/525/change-vote-breakdown-popup-hover-behaviour

https://app.shortcut.com/dux-makerdao/story/532/vertically-align-proxy-contract-label-with-address

### What does this PR do?

Remove breakdown tooltip on vote history item

Aligns proxy contract text for vote history list

### Steps for testing:

Go to `/address/0xd8bba030ae1b52a5e45f5cf05de47dcbb3a1dfc4?network=mainnet`

See that proxy contract text is now aligned and that vote history no longer has a tooltip hover

### Screenshots (if relevant):
<img width="485" alt="Screen Shot 2021-09-16 at 1 55 45 PM" src="https://user-images.githubusercontent.com/5225766/133607867-59afafaa-9159-43fd-a3cc-8808aee5cb5d.png">

### Add a GIF:
![](https://media.giphy.com/media/26FPEEgbycIv6wUYo/giphy.gif)
